src/Microsoft.ApiDesignGuidelines.Analyzers/Core/DoNotDirectlyAwaitATask.Fixer.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Analyzer.Utilities;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Editing;
+
+namespace Microsoft.ApiDesignGuidelines.Analyzers
```

Comment:
ApiDesignGuidelines is a package meant for library authors and has a bunch of "you should probably look into this" kind of rules which makes sense for libraries but are not universally appicable. We would recommend that API authors install this package only for their library projects.
Since in a library you probably should be doing this to be safely consumed by various clients, this fits that description.